README.md

```diff
@@ -20,51 +20,36 @@
 | [CrateDB]
 | [Community Forum]
 
-The `cratedb-fivetran-destination` package implements the [CrateDB destination
-adapter for Fivetran]. It works with both [CrateDB] and [CrateDB Cloud].
-
-Feel free to use the adapter as provided or else modify / extend it
-as appropriate for your own applications. We appreciate contributions of any kind.
+## About
 
-## Introduction
-
-CrateDB is a distributed and scalable SQL database for storing and analyzing
-massive amounts of data in near real-time, even with complex queries.
-It is PostgreSQL-compatible, and based on Lucene.
-
-Fivetran is an automated data movement platform. Automatically, reliably and
+[Fivetran] is an automated data movement platform. Automatically, reliably and
 securely move data from 650+ sources including SaaS applications, databases,
 ERPs, and files to data warehouses, data lakes, and more.
 
-## Requirements
+[CrateDB] is a distributed and scalable SQL database for storing and analyzing
+massive amounts of data in near real-time, even with complex queries. 
+CrateDB is based on Lucene and Elasticsearch, but [compatible with PostgreSQL].
```

Comment:
@surister: I've used the "compatible **with** PostgreSQL" jargon now, like CockroachDB is doing it, instead of saying "it **is** PostgreSQL-compatible", which may indicate a 100% compatibility, which is clearly not an applicable evaluation.

I've also minted a Hyperlink on this very detail, to have a page where we can elaborate a bit more beyond what's on the [PostgreSQL wire protocol](https://cratedb.com/docs/crate/reference/en/latest/interfaces/postgres.html) page, and also connect that with the upcoming page/section about "Data modeling".

/cc @hlcianfagna, @kneth